persistence/sql/migrations/sql/20220825134336000000_not_null_constraint_verification_token_flow_id.sqlite3.down.sql

```diff
@@ -0,0 +1,37 @@
+ALTER TABLE identity_verification_tokens
```

Comment:
How confident are you that this migration is correct and not, for example, missing some keys?

Reply:
I copied the schema + indices from the SQLite database that I ran the migrations against. So, pretty confident. 

Also, we should see this breaking something in the E2E tests, right?

Reply:
If the schema is incompatible yes, if it's like a small bug (missing index) then probalby not! So yes, some cases will be caught but some won't :/